### PR TITLE
SCSS - Create a vacancy as Stadtman Vacancy Manager

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

File: src/test/java/ServiceNow/SCSS/Pages/OWMVacancyPage.java
Comment (matakevin) [inline src/test/java/ServiceNow/SCSS/Pages/OWMVacancyPage.java:1]: line 71 should be deleted

**Review by matakevin — CHANGES_REQUESTED** [review]
Just delete that one line and should be good to go.  Thank you!

File: src/test/java/ServiceNow/SCSS/Pages/OWMVacancyPage.java
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Pages/OWMVacancyPage.java:1]: Deleted

**Review by matakevin — APPROVED** [review]
LGTM

**Review by georgeb79 — APPROVED** [review]

**Pull request merged**